iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/plugin/KubernetesParser.java

```diff
@@ -77,6 +78,8 @@ private FileTree parseHelmFile(String source, @Nullable InputFileContext inputFi
       fileRelativePath = inputFileContext.inputFile.filename();
     } else {
       fileRelativePath = chartRootDirectory.relativize(filePath).normalize().toString();
+      // transform windows to unix path
+      fileRelativePath = fileRelativePath.replace(File.separatorChar, '/');
```

Comment:
Currently this produces a lot of issues on master for windows, because the Go-Code expects unix separators. Not sure if this is the best way, but was an easy fix.

Reply:
I think it makes sense to handle all this logic on the Java side so that Go can simply operate with Unix separators and LF line breaks.